modules/src/xibo-player.js

```diff
@@ -704,6 +704,28 @@ XiboPlayer.prototype.init = function() {
   // Create global render array of functions
   window.renders = [];
 
+  // If we have scoped styles
+  // convert the CSS rules to use it
+  $('[data-style-scope]').each((_idx, styleEl) => {
```

Comment:
Does this handle a widget cached before this change? I.e. where there is no type/datatype/templateId ?